### DISCUSSION TITLE
Adding items to a comboBox

### DISCUSSION THREAD

**Discussion by AviadBly** [AvaloniaUI/Avalonia#11937] [Q&A] [answered]
Hey, Does anyone know how to add an item or delete an item from a comboBox (in the c# codebehind)? 

There's not enough documentation about this stuff :)

**Comment by timunie** [comment, marked as answer]
Your best bet is to use the MVVM approach. In your ViewModel create an `ObservableCollection`. You can add or remove items there easily. The ComboBox will pick it up as needed. 

If MVVM is new to you, do some tutorials in advance: 
- https://github.com/AvaloniaUI/Avalonia.Samples 
- https://docs.avaloniaui.net/docs/tutorials/

**Reply by AviadBly** [reply]
Thank you, will look into it
